03/script.js

```diff
@@ -0,0 +1,80 @@
+{
+     // String: greeting
+     const question = prompt('як вас звати?')
+     const answer = alert(`${question} привіт!`)
+}
+
+{
+     // String: gopni4ek
+     const words = prompt('Напишіть слова через кому')
+     const splitJoin = alert(`${words.split(',').join('блін')}`)
+}
+
+// String: capitalize
+let str = "cANBerRa"
```

Comment:
even if in task says `let` think if you really need to create variable via `let` or `const`